src/main/java/uk/gov/hmcts/reform/idam/api/fr/idm/selfservice/SelfServiceRegistrationApi.java

```diff
@@ -34,6 +34,7 @@
   @Headers({
     "Content-Type: application/json",
     "Accept: application/json",
+    "X-emailType: {emailType}"
```

Comment:
Have these changes been generated using `gradle generateApi` or manually?

Reply:
manual

Reply:
Can you please regenerate the api automatically from the yaml files (`gradle generateApi`) then copy the relevant part to the actual code from `build/src/generated`? Thanks

Reply:
yeah I think some things will change if we generate automatically, order of fields maybe, not sure if it's a big deal

Reply:
If you are talking about existing code, then that would only mean that there are manual changes there already which should not be there in the first place. But yes, the regeneration can potentially change the order of the params. Easy to spot though as tests will start failing :)

Reply:
regenerated but also modified the generated code